docs/slither.md

```diff
@@ -0,0 +1,74 @@
+Slither Integration
+============
+
+_(Supported since Remix IDE v0.15.0 and Remixd v0.4.3)_
+
+Remixd and Slither
+------------------
+
+**Note:** If you have not used `remixd` before, read more about it [here](./remixd.html)
+
+Since Remixd `v0.4.3`, when `remixd` is running locally on your device, an additional websocket plugin will be listening on port `65523` which will be dedicated for [Slither](https://github.com/crytic/slither) integration.
+
+> _Slither is a Solidity static analysis framework written in Python 3. It runs a suite of vulnerability detectors, prints visual information about contract details, and provides an API to easily write custom analyses._
+
+The remixd Slither listener is a websocket plugin similar to remixd and is used to perform Slither analysis with Remix IDE. 
+
+It doesn't need any separate installation as it is shipped with [remixd NPM](https://www.npmjs.com/package/@remix-project/remixd) module.
+
+![](images/a-slither-remixd.png)
+
+Enable Slither Analysis
+------------------
+
+### Prerequisites
+
+To use Slither analysis with Remix IDE efficiently, following tools should be installed locally on the system:
+
+1. **Slither:** [https://github.com/crytic/slither#how-to-install](https://github.com/crytic/slither#how-to-install)
+2. **Solc:** [https://docs.soliditylang.org/en/latest/installing-solidity.html](https://docs.soliditylang.org/en/latest/installing-solidity.html)
+3. **Solc-select:** [https://github.com/crytic/solc-select#quickstart](https://github.com/crytic/solc-select#quickstart)
+
+### How to use
+
+If a project is shared through remixd and `localhost` workspace is loaded in Remix IDE, there will be an extra checkbox shown in `Solidity Static Analysis` plugin with the label `Enable Slither Analysis`.
+
+![](images/a-slither-analysis.png)
+
+One can check this box to run the analysis using Slither along with the Remix. 
```

Comment:
One can check this box to run the analysis using Slither along with Remix's analysis library.
or
Check this box to run Sither along with whichever Remix's analysis module are checked.

Reply:
updated